.github/workflows/maven.yml

```diff
@@ -0,0 +1,110 @@
+name: Maven CI
+
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - '*'
+      - '!master'
+  pull_request:
+    branches:
+      - '*'
+      - '!master'
+
+jobs:
+  site:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up cache for ~./m2/repository
+        uses: actions/cache@v2.1.3
+        with:
+          path: |
+            ~/.m2/repository
+            !~/.m2/repository/org/codehaus/mojo/aspectj-maven-plugin
+          key: maven-${{ matrix.os }}-java${{ matrix.java }}-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            maven-${{ matrix.os }}-java${{ matrix.java }}-
+            maven-${{ matrix.os }}-
+      - name: Set up JDK
+        uses: actions/setup-java@v2.1.0
+        with:
+          java-version: 8
+          distribution: adopt-openj9
+
+      - name: Build with Maven
+        shell: bash
+        run: mvn --show-version --errors --batch-mode "-Prun-its,docs" clean site
+
+  site-all:
+
+    needs: site
+
+    strategy:
+      matrix:
```

Comment:
this project is about building the site, not really a plugin, then building on matrix of Maven+JDK+OS does not really make sense

what would make sense is that the build is build only once and the value would be to pushed it live from GH Action to http://www.mojohaus.org , without needing a developer to do it from his own machine

Reply:
Feel free to change this. It is intentionally pushed to upstream, I won't be able to alter it soon.